cmd/lima-guestagent/daemon_linux.go

```diff
@@ -21,6 +21,8 @@ func newDaemonCommand() *cobra.Command {
 		RunE:  daemonAction,
 	}
 	daemonCommand.Flags().Duration("tick", 3*time.Second, "tick for polling events")
+	daemonCommand.Flags().Bool("audit", true, "use audit features")
```

Comment:
Doesn’t WSL2 support audit?

Reply:
Can we remove this flag and auto detect the availability?

Reply:
> Doesn’t WSL2 support audit?

Seems like the kernel does have support:
```
$ modprobe configs ; gunzip -dc /proc/config.gz | grep AUDIT
modprobe: FATAL: Module configs not found in directory /lib/modules/5.15.90.1-microsoft-standard-WSL2
# CONFIG_AUDIT is not set
CONFIG_HAVE_ARCH_AUDITSYSCALL=y
CONFIG_AUDIT_ARCH=y
# CONFIG_KVM_MMU_AUDIT is not set
```

but the WSL2 instances are created without the necessary capability to use audit:
```
$ sudo auditctl -s
Error - audit support not in kernel
Cannot open netlink audit socket
```

> Can we remove this flag and auto detect the availability?

Sounds good, I'll do this in the next revision unless there's some other ideas on enabling audit in a WSL2 instance.

Reply:
Removed the flag in latest revision